docs/2023/DIGIT/Vehicle Tracker/updates/2023-08-04.md

```diff
@@ -4,13 +4,21 @@ author: Achintya
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [x] Added I18N via API and stored it in local storage
+- [x] Added MDMS via API and stored it in local storage
+- [x] Completed Home and Detail Info Screen
 
 ## Screenshots / Videos 
+There were some changes in the figma design so I have remade/modified some of my previous screens.
+
+| Home | Detail |
+| --- | --- |
+| ![](../static/week5/img-1.jpg) | ![](../static/week5/img-2.jpg) |
```

Comment:
just give a sensible alt names to the images, rest is good

Reply:
Ok